pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -58,14 +61,29 @@ python3Packages.buildPythonApplication rec {
   # Executable in $out/bin is a symlink to $out/share/dupeguru/run.py
   # so wrapPythonPrograms hook does not handle it automatically.
   postFixup = ''
-    wrapPythonProgramsIn "$out/share/dupeguru" "$out $pythonPath"
+    wrapPythonProgramsIn "$out/share/dupeguru" "$out $propagatedBuildInputs"
+  '';
+
+  doCheck = true;
+
+  checkInputs = [
+    black
+    distro
+    flake8
+    pytest
+    send2trash
+    tox
+  ];
+
+  checkPhase = ''
+    tox
```

Comment:
Can we execute pytest through pytestCheckHook?

Reply:
Fixed.